src/main.js

```diff
@@ -116,8 +119,13 @@ class SliderEntityRow extends LitElement {
       ha-entity-toggle {
         margin-left: 8px;
       }
-      ha-slider.full {
+      ha-slider {
         width: 100%;
+        min-width: 100px;
```

Comment:
without this is better
overflow when card width is 260~300

Reply:
I have lowered the value a bit now. Not sure if deleting the min-width is a good idea because otherwise the slider can get so small that you cannot really use it anymore.

I'm curious to hear the opinion of @thomasloven about this.